Tests/TestingTests/SwiftPMTests.swift

```diff
@@ -492,4 +500,25 @@ struct SwiftPMTests {
     let args = try parseCommandLineArguments(from: ["PATH", "--verbosity", "12345"])
     #expect(args.verbosity == 12345)
   }
+
+  @Test("--foo=bar form")
+  func equalsSignForm() throws {
+    // We can split the string and parse the result correctly.
+    do {
+      let args = try parseCommandLineArguments(from: ["PATH", "--verbosity=12345"])
+      #expect(args.verbosity == 12345)
+    }
+
+    // We don't overrun the string and correctly handle empty values.
+    do {
+      let args = try parseCommandLineArguments(from: ["PATH", "--xunit-output="])
+      #expect(args.xunitOutput == "")
+    }
+
+    // We split at the first equals-sign.
```

Comment:
If you moved these comments down to the line above each `#expect`, they'd be expectation comments in the results! 😅

Makes me wonder if we ought to look for code comments above `do { ... }` blocks in lexical scope (probably stopping at the nearest-enclosing function scope)